parm/config/gfs/config.resources

```diff
@@ -819,568 +756,524 @@ case ${step} in
       fi
       export ICEPETS ICETHREADS
 
-      echo "Total PETS for ${_CDUMP} = ${NTASKS_TOT}"
+      echo "Total PETS for ${_RUN} = ${NTASKS_TOT}"
 
-      declare -x "npe_${step}_${_CDUMP}"="${NTASKS_TOT}"
-      declare -x "nth_${step}_${_CDUMP}"="${UFS_THREADS}"
-      declare -x "npe_node_${step}_${_CDUMP}"="${npe_node_max}"
+      declare -x "ntasks_${_RUN}"="${NTASKS_TOT}"
+      declare -x "threads_per_task_${_RUN}"="${UFS_THREADS}"
+      declare -x "tasks_per_node_${_RUN}"="${max_tasks_per_node}"
```

Comment:
Similarly, I don't think we need to tag these with a `_RUN` anymore. Or the wallclock below.

Reply:
Fixed